docs/auth/tokens.rst

```diff
@@ -268,6 +268,126 @@ If you do not have the token UUID, but you do have the token value itself, you
 can use the :ref:`log-out` endpoint to delete it.
 
 
+Token Scoping: Domain Policies
+``````````````````````````````
+Tokens by default can be used to authorize any DNS operation on any domain
+within the user's account.
```

Comment:
and also for admin work, mention that here to avoid misunderstandings?

Reply:
done